vault_request_token.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"time"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/golang/glog"
+	"github.com/hashicorp/vault/api"
+)
+
+const (
+	serviceAccountTokenPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+	vaultTokenFilePath      = "/var/run/secrets/vaultproject.io/.vault-token"
+	vaultAuthFilePath       = "/var/run/secrets/vaultproject.io/.vault-auth"
+)
+
+var (
+	vaultAuthPath = getEnv("VAULT_AUTH_PATH", "auth/kubernetes")
+	vaultAddress  = getEnv("VAULT_ADDR", "http://vault.svc.cluster.local")
```

Comment:
is this a safe default given this is used by the wider community?

Reply:
As those are generic paths nothing sensitive is exposed was my thought process.